### PR TITLE
Ecf tech 37 rails upgrade

### PR DESCRIPTION
**Pull request by ethanmills**
### Context
Rails 6.1 is out and we have no code, so it looks like a good time to upgrade

### Changes proposed in this pull request
Pretty simple upgrade here, the big graft was getting some of our dependencies to support 6.1. A couple of issues and a PR later and they all do 🎉

### CODE REVIEW

File: .githooks/pre-commit [hunk 1]
```diff
@@ -6,11 +6,12 @@ git apply -R unstaged.diff
```
Comment (ethanmills) [inline .githooks/pre-commit:6]: This isn't related to the upgrade, but I needed it to be able to commit the files 😛

File: config/initializers/permissions_policy.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Define an application-wide HTTP permissions policy. For further
+# information see https://developers.google.com/web/updates/2018/06/feature-policy
+#
+# Rails.application.config.permissions_policy do |f|
+#   f.camera      :none
```
Comment (amazimbe) [inline config/initializers/permissions_policy.rb:7]: All the code has been commented out in here. Is this file needed?

**Review by amazimbe — APPROVED** [review]

File: config/initializers/permissions_policy.rb (hunk 2 above)
Comment (ethanmills) [reply config/initializers/permissions_policy.rb:7]: No, and I don't expect we'll need them in the future, but there's no harm in leaving them there in case. I think this now comes as part of the rails template
